# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/gc/bug-1941728.js
# Commit: 1055931aad82
# Full Hash: 1055931aad820878188250e89d5dacb35b893400
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2025-01-17 21:57:54
# Description:
#   Bug 1941728 - Increase the granularity of large buffer allocations to 1MB r=sfink
#   
#   Currently the granularity is PageSize, but that may be less then the system
#   page size.
#   
# ==============================================================================

diff -r f10cdccac853 -r 1055931aad82 js/src/jit-test/tests/gc/bug-1941728.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/gc/bug-1941728.js	Fri Jan 17 15:23:05 2025 +0000
@@ -0,0 +1,4 @@
+var x = [];
+for (let i = 0; i < 9999999; i++) {
+  x.push(0);
+}